modules/ROOT/pages/dataweave-language-introduction.adoc

```diff
@@ -301,33 +302,62 @@ The following table provides examples of invalid identifiers and a description o
 
 Valid identifiers cannot match any of the reserved DataWeave keywords:
 
-* `and`
-* `as`
-* `async`
-* `case`
-* `default`
-* `do`
-* `else`
-* `enum`
-* `false`
-* `for`
-* `fun`
-* `if`
-* `import`
-* `input`
-* `is`
-* `ns`
-* `null`
-* `or`
-* `output`
-* `private`
-* `throw`
-* `true`
-* `type`
-* `unless`
-* `using`
-* `var`
-* `yield`
+* `and`: +
```

Comment:
I would GLOBALLY remove the colons and the periods following the introductory fragment, moving the sentence to a subsequent line. See first couple of example markups below:

Reply:
```suggestion
* `and` +
```